Marlin/Marlin_main.cpp

```diff
@@ -5461,7 +5516,13 @@ void home_all_axes() { gcode_G28(true); }
               #if ENABLED(PROBE_MANUALLY)
                 z_at_pt[axis] += lcd_probe_pt(cos(a) * r, sin(a) * r);
               #else
-                z_at_pt[axis] += probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1, false);
+                z_at_pt[axis] += probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1);
+                if (isnan(z_at_pt[axis])) {
+                  LCD_MESSAGEPGM(MSG_ERR_PROBING_FAILED);
+                  SERIAL_ERROR_START();
+                  SERIAL_ERRORLNPGM(MSG_ERR_PROBING_FAILED);
```

Comment:
The repetition of these three lines indicates potential design flaw. Probably this whole function needs to be refactored into some smaller functions. At the least, this set of three lines should be a separate function.